### PR TITLE
Update action versions to remove warnings

### PR DESCRIPTION
**Pull request by thomasm-ttd**

### CODE REVIEW

File: .github/workflows/shared-publish-to-docker-versioned.yaml [hunk 1]
```diff
@@ -72,6 +72,6 @@ jobs:
           is_release: ${{ steps.setup.outputs.is_release }}
           new_version: ${{ inputs.new_version }}
           tags: ${{ steps.publishToDocker.outputs.tags }}
-          image_tag: ${{ steps.updatePackageJson.outputs.image_tag }}
+          image_tag: ${{ inputs.docker_image_name }}
```
Comment (Ian-Nara) [inline .github/workflows/shared-publish-to-docker-versioned.yaml:75]: just curious, what is this change for?

Comment (thomasm-ttd) [reply .github/workflows/shared-publish-to-docker-versioned.yaml:75]: The updatePackageJson doesn't exist

Comment (cYKatherine) [reply .github/workflows/shared-publish-to-docker-versioned.yaml:75]: Interesting ...how the pipeline passed before 🤔

**Review by cYKatherine — APPROVED** [review]

File: .github/workflows/shared-publish-to-docker-versioned.yaml (hunk 1 above)
Comment (thomasm-ttd) [reply .github/workflows/shared-publish-to-docker-versioned.yaml:75]: If a variable is not found, it is just empty

**Pull request merged**